src/App/tests/__snapshots__/App.test.tsx.snap

```diff
@@ -558,74 +558,6 @@ exports[`App renders App correctly 1`] = `
                                       </NavLink>
                                     </li>
                                   </NavItem>
-                                  <NavItem
```

Comment:
<img width="1436" alt="Screen Shot 2019-10-03 at 9 25 42 PM" src="https://user-images.githubusercontent.com/4141527/66181121-7ead2b80-e624-11e9-9f20-7e91fc2a2b68.png">

For some reason the global navigation snapshots are failing, regardless of local env variables at the time of testing (see how both b936e43 vs c7c7560 fail in screenshot).

Reply:
@cKellyDesign I am continuing work on this pr on smaller prs, and they work is quite diverged from the one that is on here. That being said, the snapshot tests seem to fail because of mismatches in the value of the env variables: `REACT_APP_ENABLE_USERS=true
REACT_APP_ENABLE_ABOUT=true`

Reply:
I've tried setting those variables as true and true and still the test (on the left) was failing.